schematics/utils/ast.ts

```diff
@@ -0,0 +1,57 @@
+import { SchematicsException } from '@angular-devkit/schematics';
+import { Tree } from '@angular-devkit/schematics';
+import * as ts from 'typescript';
+import { addImportToModule } from './devkit-utils/ast-utils';
+import { getAppModulePath } from './devkit-utils/ng-ast-utils';
+import { InsertChange } from './devkit-utils/change';
+import { getConfig, getAppFromConfig } from './devkit-utils/config';
+import { normalize } from '@angular-devkit/core';
```

Comment:
Omit spaces in braces (WebStorm can be configured for this, not sure about VS Code)

(here and elsewhere)